packages/web/src/pages/upload-page/components/ReleaseDateModalField.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react'
+
+import {
+  Button,
+  ButtonType,
+  IconCalendar,
+  IconCaretRight,
+  Modal,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalTitle
+} from '@audius/stems'
+import cn from 'classnames'
+import { useField } from 'formik'
+
+import { DatePickerField } from '../fields/DatePickerField'
+
+import styles from './ReleaseDateModalField.module.css'
+
+const messages = {
+  title: 'Release Date',
+  description:
+    'Release date affects sorting on your profile and is visible in track details.',
+  done: 'Done'
+}
+
+const FIELD_NAME = 'releaseDate'
+
+export const ReleaseDateModalField = () => {
+  const [{ value }] = useField<moment.Moment>(FIELD_NAME)
+  const [isModalOpen, setIsModalOpen] = useState(false)
+  const open = () => setIsModalOpen(true)
+  const close = () => setIsModalOpen(false)
+
+  const modal = (
+    <div className={styles.modal}>
+      <Modal onClose={close} isOpen={isModalOpen}>
+        <ModalHeader>
+          <div className={styles.modalHeader}>
```

Comment:
nit: are we able to add this class name to the ModalHeader instead of adding another div?